src/sentry/event_manager.py

```diff
@@ -1801,6 +1801,9 @@ def _handle_regression(group: Group, event: Event, release: Optional[Release]) -
     group.active_at = date
     group.status = GroupStatus.UNRESOLVED
     group.substatus = GroupSubStatus.REGRESSED
+    # we already update these fields in the DB on .update() call, but we need to .save()
+    # to ensure the cache is updated properly since post_process_group retrieves the cached group
+    group.save(update_fields=["active_at", "status", "substatus"])
```

Comment:
Are you using `Model.update` or `Model.objects.update`? I'd expect the former to fire signals.

Wondering if we should just manually bust the cache instead? Or manually fire the signal?

Reply:
Right, we definitely do `Model.objects.update` here https://github.com/getsentry/sentry/blob/edafbfbed7eb38737a41ecce82bdec61e0ca3a8f/src/sentry/event_manager.py#L1773-L1791

Yeah, I think firing the signal is probably better.